index.d.ts

```diff
@@ -1,8 +1,8 @@
 declare namespace hubdown {
   interface Options {
-    readonly runBefore: Array<any>;
-    readonly frontmatter: boolean;
-    readonly cache: any;
+    readonly runBefore?: Array<any>;
+    readonly frontmatter?: boolean;
+    readonly cache?: any;
```

Comment:
Not available yet, but https://github.com/Level/level/issues/120 may someday offer a type the could be used here.